app/src/main/res/values/preferences.xml

```diff
@@ -123,6 +123,12 @@
     <string name="preference_title_insistent_alarms_enabled">Enable insistent alarm</string>
     <string name="preference_summary_insistent_alarms_enabled">Keeps an alarm on infinitely.</string>
 
+    <!-- User Interface -->
+    <string name="preference_key_orientation_schedule_landscape" translatable="false">user_interface_landscape</string>
+    <bool name="preference_default_orientation_schedule_landscape">false</bool>
+    <string name="preference_title_orientation_schedule_landscape">Schedule in landscape mode</string>
+    <string name="preference_summary_orientation_schedule_landscape">Show schedule UI always in landscape mode independent of device settings.</string>
```

Comment:
Here I would expect the title to be `Force landscape mode` and the summary to be `Shows all screens always in landscape mode independent of the device settings.` (third person to match the wording of the other settings) now that not only the schedule screen is affected.

Please rename the preferences items accordingly to `..._force_landscape_mode_enabled`.